python/katana_setup.py

```diff
@@ -349,9 +350,22 @@ def _build_cython_extensions(pyx_files, source_root_name, extension_options):
 def cythonize(module_list, *, source_root, **kwargs):
     # TODO(amp): Dependencies are yet again repeated here. This needs to come from a central deps list.
     require_python_module("packaging")
-    require_python_module("Cython", "0.29.12")
     require_python_module("numpy", "1.10")
-    require_python_module("pyarrow", "1.0", "3.0.dev")
+    try:
+        require_python_module("Cython", "0.29.12")
+        require_python_module("pyarrow", "4.0", "5.0.dev")
+        build_extensions = True
+    except RequirementError:
+        print(
```

Comment:
used "import warnings" in init and print here

Reply:
I did this because `warnings` is for Python warnings (like exceptions, but non-fatal) where as this is a build warning really. That said I think you might be right. However I'm gonna leave this as is. This entire file uses `print` so I'll stick to that in this PR.